src/test/java/ruslan/shastkiv/bookstore/service/BookServiceTest.java

```diff
@@ -70,12 +76,10 @@ void createBook_ValidCreateBookDto_ReturnBookDto() {
         CreateBookRequestDto requestDto = createBookRequestDtoById(FIRST_BOOK_ID);
         Book book = createBookById(FIRST_BOOK_ID, List.of(FIRST_CATEGORY_ID));
         BookDto expectedBookDto = createBookDtoById(FIRST_BOOK_ID, List.of(FIRST_CATEGORY_ID));
-        when(bookMapper.toModel(requestDto)).thenReturn(book);
+
         when(categoryRepository.findAllById(Set.of(FIRST_CATEGORY_ID)))
                 .thenReturn(List.of(createCategoryById(FIRST_CATEGORY_ID)));
-        when(bookRepository.save(book)).thenReturn(book);
-        when(bookMapper.toDto(book)).thenReturn(expectedBookDto);
-
+        when(bookRepository.save(any(Book.class))).thenReturn(book);
```

Comment:
As a best practice, I recommend not any() argument matches, it's better to use exact values when stubbing. The only case when any() is a good tool for you - is when there is no way to use exact matching :)